src/workerd/io/compatibility-date.capnp

```diff
@@ -312,4 +315,7 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # Perform additional error checking in the Web Compression API and throw an error if a
   # DecompressionStream has trailing data or gets closed before the full compressed data has been
   # provided.
+
+  # If you are adding a new flag, you must update the official Cloudflare docs:
+  # https://github.com/cloudflare/cloudflare-docs/blob/production/content/workers/platform/compatibility-dates.md
```

Comment:
That'd be awesome - I guess the groundwork is there with workers-types' generation being aware of compatibility dates & the scaffolding is all YAML (https://github.com/cloudflare/cloudflare-docs/tree/production/content/workers/_partials/_platform-compatibility-dates) since compatibility dates are per-file with standard fields, other than the description.

Reply:
Talked with @kflansburg, the juice may not be worth the squeeze on adding a github action. Also @irvinebroque should be in the loop here as well.

Reply:
Yeah I suspect the effort to build this would be much greater than the time it ultimately saves. I think we just need reviewers to remember to request a doc PR before approving a new compat flag...